uefi-raw/src/table/runtime.rs

```diff
@@ -0,0 +1,94 @@
+//! UEFI services available at runtime, even after the OS boots.
+
+use crate::{guid, Guid};
+use bitflags::bitflags;
+
+/// The type of system reset.
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+#[repr(u32)]
+pub enum ResetType {
+    /// Resets all the internal circuitry to its initial state.
+    ///
+    /// This is analogous to power cycling the device.
+    Cold = 0,
+    /// The processor is reset to its initial state.
+    Warm,
+    /// The components are powered off.
+    Shutdown,
+    /// A platform-specific reset type.
+    ///
+    /// The additional data must be a pointer to
+    /// a null-terminated string followed by an UUID.
+    PlatformSpecific,
+    // SAFETY: This enum is never exposed to the user, but only fed as input to
```

Comment:
This enum is now exposed to the user, isn't it?

Reply:
The enum was already public -- I think by "never exposed" the comment means "never output from a firmware function".

However, I just grepped the spec and there is actually a place where it's used in an output parameter (`QueryCapsuleCapabilities`), so I'll change this to a newtype enum.

Reply:
nice, thanks! :)